tools/vpt/macros.xml

```diff
@@ -0,0 +1,65 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.3.0</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@PROFILE@">23.0</token>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">quay.io/bgruening/vpt:1.3.0-1</container>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="creator">
+        <creator>
+            <organization name="European Galaxy Team" url="https://galaxyproject.org/eu/" />
+            <person givenName="Amirhossein" familyName="Naghsh Nilchi" email="nilchia@informatik.uni-freiburg.de" />
+            <person givenName="Pavan" familyName="Videm" email="videmp@informatik.uni-freiburg.de" />
+            <person givenName="Björn" familyName="Grüning" email="gruening@informatik.uni-freiburg.de" />
+        </creator>
+    </xml>
+    <token name="@CMD@"><![CDATA[
+        mkdir -p 'input/images' 'output/' &&
+        #for $image in $input_images:
+            ln -s '$image' 'input/images/${image.element_identifier}.${image.ext}' &&
+        #end for
+        ln -s '$input_micron_to_mosaic' 'input/micron_to_mosaic_pixel_transform.csv' &&
+        ]]></token>
+    <token name="@COMMON_ARGS@"><![CDATA[
+        --processes \${GALAXY_SLOTS:-10}
+        --verbose
+        --log-file 'output/log'
+    ]]>
+    </token>
+    <xml name="common_input">
+        <param argument="--input_images" type="data" format="tiff" multiple="true" label="MEROSCOPE tiff images"/>
+        <param argument="--input_micron_to_mosaic" type="data" format="csv" label="Micron to mosaic mapping file"/>
+    </xml>
+    <xml name="advanced_output">
+        <section name="advanced_output" title="Advanced output options">
+            <param name="log" type="boolean" truevalue="True" falsevalue="False" checked="true" label="Output log?"/>
```

Comment:
Do you plan to have more options in the future? Otherwise a section with only one parameter might be overkill.